src/js/components/List.js

```diff
@@ -149,3 +163,7 @@ List.propTypes = {
     PropTypes.arrayOf(PropTypes.number)
   ])
 };
+
+List.defaultProps = {
+  role: 'list'
```

Comment:
I like the idea of having a default prop for role. but we still need the transfer props. 

I'm ok with leaving role prop if we decide to keep default props.